content/point-data/point-data.md

```diff
@@ -255,6 +258,63 @@ Textures also work with color maps. In that case, the color of the texture is mu
 The list of available path tokens and their corresponding locations are found in the openspace.cfg file.
 :::
 
+### Other Texture Settings
+There are some additional settings that might be set related to the texture, like whether image compression should be allowed or if the alpha channel of images should be used (both these settings are enabled per default).  For more details, see the documentation of the table parameters for the `Texture` table of `RenderablePointCloud` on the [reference page about that renderable](/generated/asset-components/RenderablePointCloud.md).
+
+## Outlines
+
+:::{figure} outline_basic.png
+:align: right
+:width: 20em
+Blue points with white outline.
+:::
+
+When rendering points without additive blending, it might be difficult to distinguish the edges of the individual points. In this case, it might be useful to render a distinctly colored outline around the point.
+
+To add an outline to your point cloud, set the `Coloring.EnableOutline` setting to `true`, and possibly add an outline color and/or width setting. See example:
+
+```lua
+ ...
+  Renderable = {
+    ...
+    Coloring = {
+      FixedColor = { 0.5, 0.8, 1.0 }, -- light blue
+      -- Outline settings
+      EnableOutline = true,
+      OutlineColor = { 1.0, 1.0, 1.0 }, -- white
+      OutlineWidth = 0.1
+    }
+  },
+  ...
+```
+
+### Outline Style and Color From Color Map
```

Comment:
```suggestion
### Outline Style and Color from Color Map
```

Reply:
![image](https://github.com/OpenSpace/OpenSpace-Docs/assets/8808894/2eb7616a-1740-4fca-b282-fdf541cce273)

Which style are we using?